shenyu-bootstrap/src/main/resources/application-local.yml

```diff
@@ -144,4 +144,9 @@ logging:
     org.apache.shenyu.bonuspoint: info
     org.apache.shenyu.lottery: info
     org.apache.shenyu: info
+    # reset the com.alibaba log
+    com.alibaba.cloud.dubbo.autoconfigure.DubboServiceDiscoveryAutoConfiguration: warn
```

Comment:
we have this jar?

Reply:
when using the spring cloud Alibaba the jar exists. the config is remove the many warning message